Bottom: f39b3f5fc9935e2d0b0ad67482e4163bccc77317
Top:    b01637dd203e1c61bf333861e839e00ef89fbd38
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:36:01 +0000

Refresh of well-on-the-way-to-completely

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index c2df099..8440a48 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -28,16 +28,10 @@
 #define EPSILON 1e-5 /* XXX: Unknown  what this needs to be */
 
 #ifdef REVERSED_PCB_CONTOURS
-#define COORD_TO_STEP_X(pcb, x) (COORD_TO_MM(                   (x)))
-#define COORD_TO_STEP_Y(pcb, y) (COORD_TO_MM((pcb)->MaxHeight - (y)))
-
 #define STEP_X_TO_COORD(pcb, x) (MM_TO_COORD((x)))
 #define STEP_Y_TO_COORD(pcb, y) ((pcb)->MaxHeight - MM_TO_COORD((y)))
 #else
 /* XXX: BROKEN UPSIDE DOWN OUTPUT */
-#define COORD_TO_STEP_X(pcb, x) (COORD_TO_MM((x)))
-#define COORD_TO_STEP_Y(pcb, y) (COORD_TO_MM((y)))
-
 #define STEP_X_TO_COORD(pcb, x) (MM_TO_COORD((x)))
 #define STEP_Y_TO_COORD(pcb, y) (MM_TO_COORD((y)))
 #endif
